src/main/java/controllers/exercise03/StudentController.java

```diff
@@ -0,0 +1,108 @@
+package controllers.exercise03;
+
+import com.google.gson.reflect.TypeToken;
+import entities.exercise03.Student;
+import utils.FileUtils;
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+public class StudentController {
+    private static File file;
+
+    private static List<Student> students;
+
+    private static int COUNT;
```

Comment:
Chỉ viết hoa toàn bộ nếu là constant

```suggestion
    private static int count;
```

Reply:
done